From ea84525bff9e6869d7881d2da463f0a981a0c5ff Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 19 Mar 2019 13:05:04 +0800
Subject: [PATCH 607/706] arm64: dts: ls1028a: fix the wrong name for ls1028a
 for spi

nxp,lx2160a-fspi is not helpful used in ls1028a dtsi, so replace it
with ls1028a because there is a real SoC with lx2160a.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index e568e283a750..1c73c47aae87 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -194,7 +194,7 @@
 	};
 
 	fspi: flexspi@20c0000 {
-		compatible = "nxp,lx2160a-fspi", "simple-bus";
+		compatible = "nxp,ls1028a-fspi", "simple-bus";
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x0 0x20c0000 0x0 0x10000>,
-- 
2.17.1

